js/PaymentRequest/index.js

```diff
@@ -14,6 +14,7 @@ import type {
   PaymentShippingType,
   PaymentDetailsIOS,
   PaymentDetailsIOSRaw,
+  PaymentMethod
```

Comment:
The `PaymentMethod` type only contains one attribute: a payment method type string.  Maybe it would be simpler to just handle that information as a string without wrapping it in an object.

Reply:
it matches the response body of the new API, so we need to have a object that includes attribute paymentMethodType